src/server.rs

```diff
@@ -252,6 +261,7 @@ async fn create_relay_connection_(
         ..Default::default()
     });
     stream.send(&msg_out).await?;
+    mac_wakeup();
```

Comment:
Why don't you put `mac_wakeup` in `create_tcp_connection`?

Reply:
sorry almost blind